### PR TITLE
Add support for Scala 2.12

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Needed to move porter to 2.12

### CODE REVIEW

**Review by alexduf — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -56,7 +57,8 @@ lazy val play24 = project
   .settings(
     name := "configuration-magic-play2.4",
```
Comment (TBonnin) [inline build.sbt]: The name is not matching anymore

**Review by TBonnin — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -94,13 +94,13 @@ Thus your EC2 instance profile will need the following permissions:
 
 ## configuration-magic-play2-4
```
Comment (TBonnin) [inline README.md]: Title should be changed as well 😼

File: build.sbt [hunk 3]
```diff
@@ -50,13 +51,14 @@ lazy val core = project
     testQuick in Test <<= (testQuick in Test).dependsOn(DynamoDBLocal.Keys.startDynamoDBLocal)
   )
 
-lazy val play24 = project
+lazy val play26 = project
   .dependsOn(core)
   .settings(sharedSettings:_*)
   .settings(
-    name := "configuration-magic-play2.4",
+    name := "configuration-magic-play2.6",
```
Comment (TBonnin) [inline build.sbt:58]: @alexduf what are the consequence of changing the lib name?

Comment (alexduf) [reply build.sbt:58]: Aside from potential confusion, can't think of anything.

File: README.md (hunk 2 above)
Comment (regiskuckaertz) [reply README.md]: 😂  good catch

File: build.sbt (hunk 3 above)
Comment (regiskuckaertz) [reply build.sbt:58]: Would _not_ changing the name be more confusing?

Comment (alexduf) [reply build.sbt:58]: yeah it wasn't my smartest move :D
Do rename it though

**Review by TBonnin — APPROVED** [review]

**Pull request merged**